test/config/AppConfigSpec.scala

```diff
@@ -80,6 +80,12 @@ class AppConfigSpec extends SpecBase with ShouldMatchers {
     }
   }
 
+  "isHomePageLinksEnabled" should {
+    "return the correct value" in new Setup {
```

Comment:
do we need a test where we only show the link if this is true?

Reply:
Because we are simply ensuring just that the default value of isHomePageLinksEnabled is false and there’s no complex logic for it, i think this test is sufficient enough just to check for correct flag value